packages/create-cloudflare/e2e-tests/helpers.ts

```diff
@@ -39,7 +40,15 @@ export const runC3 = async ({
 	promptHandlers = [],
 	outputPrefix = "",
 }: RunnerConfig) => {
-	const proc = spawn("node", ["./dist/cli.js", ...argv]);
+	const cmd = "node";
+	const args = ["./dist/cli.js", ...argv];
+	const proc = spawn(cmd, args);
+
+	console.log(
+		bgBlue(
+			`${outputPrefix} Running C3 with command: \`${cmd} ${args.join(" ")}\``
```

Comment:
This makes things a lot easier, can't believe we didn't add this sooner!

How about adding `TEST_PM=${process.env.TEST_PM}` to the beginning?

Reply:
Seems like it works to me?

![image](https://github.com/cloudflare/workers-sdk/assets/204386/e69191d9-7f40-4deb-8de8-431dc828d617)

Reply:
yeah, because I updated it not to use chalk 😅

https://github.com/cloudflare/workers-sdk/blob/a7399e80f6bcb88ed329df62ebec8fa4c5bd1f31/packages/create-cloudflare/e2e-tests/helpers.ts#L51-L55

what do you think? is this ok? 😅